vignettes/04-synthesis-data.Rmd

```diff
@@ -1 +1,150 @@
-# Synthesis Vignette
\ No newline at end of file
+# Combining trait, weather, and image datasets
+
+The objective of this vignette is to walk through how to combine our several types of data, and demonstrate several realistic analyses that can be done on these merged data. 
+
+For the first analysis, we want to figure out how the number of sufficiently warm days affects the amount of canopy cover at our site. We do this by combining the canopy cover data with the meteorological data on growing degree days, then modeling and plotting their relationship. We are specifically interested in figuring out when the increase in canopy cover starts to slow down in response to warm temperature days. 
+
+The second analysis compares greenness from image data with canopy cover. 
+
+## Get and join data
+
+Here we combine two dataframes. 
+The first contains all the canopy height values for 2017, which was created in the traits vignette. 
+The second is the cumulative growing degree days for all of 2017, which were calculated from the daily minimum and maximum temperatures in the weather vignette. 
+They are combined by their common column, the date. 
+
+```{r synth_setup}
+library(dplyr)
+library(ggplot2)
+library(jsonlite)
+library(lubridate)
+library(traits)
+library(inflection)
+
+options(betydb_url = "https://terraref.ncsa.illinois.edu/bety/",
+        betydb_api_version = 'v1', 
+        betydb_key = '9999999999999999999999999999999999999999')
+```
+
+```{r get_trait_data, message = FALSE}
+trait <- betydb_query(table     = "search", 
+                      trait     = "canopy_height", 
+                      date      = "~2017",
+                      limit     =  "none")
+
+trait = trait %>% 
```

Comment:
```suggestion
trait <- trait %>% 
```

Reply:
Sorry, I never got in the habit of using the weird arrow thing because I learned Python at the same time as R. I'll make sure to go back and change all of those.

Reply:
would the keyboard shortcut "option" + "-" help?

Reply:
I do use that shortcut, I just need to make it a habit.